apps/parsley/src/analytics/sectionsFeatureDiscovery/useSectionsFeatureDiscoveryAnalytics.ts

```diff
@@ -0,0 +1,16 @@
+import { useAnalyticsRoot } from "@evg-ui/lib/analytics/hooks";
+import { AnalyticsIdentifier } from "analytics/types";
+
+type Action =
+  | {
+      name: "Clicked sections feature modal cancel button";
+      release: "beta";
```

Comment:
beta will be replaced by production when we switch to the full release

Reply:
Since this value is consistent is it necessary? We can tag feature releases on honeycomb dashboards if the goal  is to track adoption before and after the release

Reply:
This feature modal will be shown again with slightly different copy for the production release. I planned on setting `release` will be set to `production` when that component is implemented. Maybe we can use the release tag in honeycomb so all other events are covered.